src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/Transmission/OtlpExporterTransmissionHandler.cs

```diff
@@ -0,0 +1,108 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#nullable enable
+
+using OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.ExportClient;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.Transmission;
+
+internal class OtlpExporterTransmissionHandler<TRequest>
+{
+    public OtlpExporterTransmissionHandler(IExportClient<TRequest> exportClient)
+    {
+        Guard.ThrowIfNull(exportClient);
+
+        this.ExportClient = exportClient;
+    }
+
+    protected IExportClient<TRequest> ExportClient { get; }
+
+    /// <summary>
+    /// Sends export request to the server.
+    /// </summary>
+    /// <param name="request">The request to send to the server.</param>
+    /// <returns>True if the request is sent successfully or else false.</returns>
+    public bool SubmitRequest(TRequest request)
```

Comment:
```suggestion
    public bool TrySubmitRequest(TRequest request)
```

Reply:
For my understanding, Why would `TrySubmitRequest` be more apt here?

Reply:
Because it's trying to submit the request and may or may not succeed in doing that. That's why it returns a bool to indicate if the request was indeed submitted. This is my understanding of what this method is doing. Correct me if I'm wrong.